charts/sf-presto/values.yaml

```diff
@@ -95,13 +102,3 @@ terminationGracePeriodSeconds: 30
 nodeSelector: {}
 tolerations: []
 affinity: {}
-imagePullSecrets: []
```

Comment:
why this is removed?

Reply:
there were duplicate entries for this key